megamek/src/megamek/client/bot/princess/FireControl.java

```diff
@@ -334,10 +334,10 @@ ToHitData guessToHitModifierHelperForAnyAttack(final Entity shooter,
 
         if (shooter instanceof Mech) {
             final Mech shooterMech = (Mech) shooter;
-            if (Mech.COCKPIT_INDUSTRIAL == shooterMech.getCockpitType()) {
-                toHitData.addModifier(TH_INDUSTRIAL);
-            } else if (Mech.COCKPIT_PRIMITIVE_INDUSTRIAL == shooterMech.getCockpitType()) {
+            if (Mech.COCKPIT_PRIMITIVE_INDUSTRIAL == shooterMech.getCockpitType()) {
```

Comment:
It looks like this change requires an update to the test `testGuessToHitModifierHelperForAnyAttack` in
`FireControlTest.java:1335`